.github/workflows/ci.yml

```diff
@@ -2,13 +2,17 @@ name: build
 
 on:
   pull_request:
+  push:
 
 jobs:
   ci:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
 
+      - name: Lint
```

Comment:
@ezio-melotti may you remove this? We can provide this linter in our tool set but I don't think it is ready for production use.

Reply:
I now removed this.  Let me know if there is anything else I should do before we merge this.  You can also ping me on Discord if you want to talk about what should be changed in the workflow and/or `sphinx-lint` to make it suitable for this repo.